testing/s3mock/src/test/java/org/projectnessie/s3mock/TestIcebergS3MockServer.java

```diff
@@ -132,6 +132,7 @@ public int doGetStatus(URI uri, Map<String, String> headers) throws Exception {
     return ((HttpURLConnection) conn).getResponseCode();
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Call to `createServer` in this class seem to deserve a `@SuppressWarnings("resource")`.